lambdas/db-migration/src/index.ts

```diff
@@ -16,30 +18,54 @@ const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
   throw new Error(`The ${name} environment variable must be set`);
 };
 
-const getConnectionConfig = (env: NodeJS.ProcessEnv): Knex.PgConnectionConfig => ({
-  host: getRequiredEnvVar('PG_HOST', env),
-  user: getRequiredEnvVar('PG_USER', env),
-  // TODO Get this value from secrets manager
-  password: getRequiredEnvVar('PG_PASSWORD', env),
-  database: getRequiredEnvVar('PG_DATABASE', env),
-});
+const getSecretConnectionConfig = async (SecretId: string): Promise<Knex.PgConnectionConfig> => {
```

Comment:
It seems like we're going to need this in several lambdas, so maybe we can put this in the `db` package?

Also, we have this if we want to dogfood our own aws-client package, but I'm not sure if it really improves anything

https://github.com/nasa/cumulus/blob/master/packages/aws-client/src/SecretsManager.ts#L7

Reply:
Maybe.  I subconsciously hoping to avoid Core entanglements in this specific case, as for the moment the aws-client package doesn't really offer much here, but that's not really a good justification for not drying it up.

Reply:
The one thing about using the aws-client package is that it automatically switches to Localstack in unit test environments. That probably doesn't matter in this case, I assume.

In any case, I care less about using `aws-client`, but if we're going to copy `getSecretConnectionConfig` across multiple lambdas, it seems like it would be nice if it lived somewhere that it could be shared? Is there some extra complexity that creates that we don't want?